langwatch/prisma/migrations/20260728120000_process_manager_inbox_digest_key/migration.sql

```
@@ -0,0 +1,46 @@
+UPDATE "ProcessManagerInbox"
+SET "sourceEventKey" = encode(sha256(convert_to("sourceEventId", 'UTF8')), 'hex')
+WHERE "sourceEventKey" IS NULL;
+
+-- Swap the constraint. The old index cannot have held a row over the btree
+-- limit (that was exactly the failure), so the digest of every surviving row is
+-- as unique as the raw value it came from and the new index cannot conflict.
+--
+-- Guarded both ways to match the migration that created the old index
+-- (20260716140000 used CREATE UNIQUE INDEX IF NOT EXISTS), so an environment
+-- that diverged does not fail this one mid-transaction.
+DROP INDEX IF EXISTS "ProcessManagerInbox_processName_projectId_sourceEventId_key";
```

SQLFluff (4.2.2)

[error] DROP statement should use CONCURRENTLY to avoid locking the table.

(PG01)